pom.xml

```diff
@@ -6,26 +6,55 @@
 
     <groupId>com.datastax.examples</groupId>
     <artifactId>storing-and-retrieving-blobs</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-core</artifactId>
+            <version>4.3.0</version>
```

Comment:
There is a newer version if you want to reference this one for sake of latest.  
<driver.version>4.5.1</driver.version>
We should verify that with Chris just in case.

Reply:
Doh, thanks. I meant to check that.